Parser/pegen/pegen.c

```diff
@@ -178,12 +178,44 @@ tokenizer_error(Parser *p)
         case E_EOFS:
         case E_EOLS:
             return tokenizer_string_error(p);
+        case E_INTR:
+            if (!PyErr_Occurred()) {
+                PyErr_SetNone(PyExc_KeyboardInterrupt);
+            }
+            return -1;
+        case E_NOMEM:
+            PyErr_NoMemory();
+            return -1;
+        case E_TABSPACE:
+            errtype = PyExc_TabError;
+            msg = "inconsistent use of tabs and spaces in indentation";
+            break;
+        case E_OVERFLOW:
+            msg = "expression too long";
+            break;
+        case E_DEDENT:
+            errtype = PyExc_IndentationError;
+            msg = "unindent does not match any outer indentation level";
+            break;
+        case E_TOODEEP:
+            errtype = PyExc_IndentationError;
+            msg = "too many levels of indentation";
+            break;
+        case E_DECODE: {
+            msg = "unknown decode error";
+            break;
+        }
+        case E_LINECONT:
+            msg = "unexpected character after line continuation character";
+            break;
+        case E_BADSINGLE:
+            msg = "multiple statements found while compiling a single statement";
+            break;
```

Comment:
Related to #99. We currently do not check for this, although we should, since this a parser issue and not a tokenizer issue.

Reply:
Yep, this should not be checked here I think

Reply:
It's not exactly clear what you mean by "here". You mean that it should be checked by the tokenizer or something else?

Reply:
> You mean that it should be checked by the tokenizer or something else?

Yup: I wanted to say that `E_BADSINGLE` will never be reached here and as you mentioned should be handled by the parser so we should exclude it from the `case`.